runtime/src/main.rs

```diff
@@ -70,16 +66,6 @@ extern "C" fn exception_handler(trap_record: &TrapRecord) {
         trap_record.ra,
     );
 
-    {
-        let mut soc_ifc = unsafe { SocIfcReg::new() };
-        let soc_ifc = soc_ifc.regs_mut();
-        let ext_info = soc_ifc.cptra_fw_extended_error_info();
-        ext_info.at(0).write(|_| trap_record.mcause);
-        ext_info.at(1).write(|_| trap_record.mscause);
-        ext_info.at(2).write(|_| trap_record.mepc);
-        ext_info.at(3).write(|_| trap_record.ra);
-    }
```

Comment:
Should we also call `log_trap_record` here? (And same in FMC exception handler)

Reply:
done.